pages/common/wami.md

```diff
@@ -0,0 +1,20 @@
+# wami
+
+> A tool that recommends suitable programs for specific tasks. Open-source, easy to use.
+> More information: <https://github.com/evait-security/wami>.
+
+- Search the lake and find expanded results in all categories in descending order:
+
+`wami -a -S desc -s {{search-string}}`
```

Comment:
[The names of placeholders cannot contain hyphens](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).

```suggestion
`wami -a -S desc -s {{search_string}}`
```